crates/tauri-utils/src/config.rs

```diff
@@ -2941,11 +2941,11 @@ fn default_min_sdk_version() -> u32 {
 #[serde(untagged, deny_unknown_fields)]
 #[non_exhaustive]
 pub enum FrontendDist {
-  /// An external URL that should be used as the default application URL.
+  /// An external URL that should be used as the default application URL. No assets are embed to the app in this case.
```

Comment:
```suggestion
  /// An external URL that should be used as the default application URL. No assets are embedded in the app in this case.
```
wanted to replace "are" with "will be" but that gets a bit long